evaluation/webarena/scripts/run_infer_2.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# configure webarena websites
+export SHOPPING="http://ec2-18-219-239-190.us-east-2.compute.amazonaws.com:7770/"
+export SHOPPING_ADMIN="http://ec2-18-219-239-190.us-east-2.compute.amazonaws.com:7780/admin"
+export REDDIT="http://ec2-18-219-239-190.us-east-2.compute.amazonaws.com:9999"
+export GITLAB="http://ec2-18-219-239-190.us-east-2.compute.amazonaws.com:8023"
+export WIKIPEDIA="http://metis.lti.cs.cmu.edu:8888/wikipedia_en_all_maxi_2022-05/A/User:The_other_Kiwix_guy/Landing"
+export MAP="http://miniserver1875.asuscomm.com:3000"
+export HOMEPAGE="http://metis.lti.cs.cmu.edu:4399"
+
+MODEL_CONFIG=$1
+AGENT=$2
+EVAL_LIMIT=$3
+
+if [ -z "$AGENT" ]; then
+  echo "Agent not specified, use default BrowsingAgent"
+  AGENT="BrowsingAgent"
+fi
+
+# IMPORTANT: Because Agent's prompt changes fairly often in the rapidly evolving codebase of OpenDevin
+# We need to track the version of Agent in the evaluation to make sure results are comparable
+AGENT_VERSION=v$(poetry run python -c "import agenthub; from opendevin.controller.agent import Agent; print(Agent.get_cls('$AGENT').VERSION)")
+
+echo "AGENT: $AGENT"
+echo "AGENT_VERSION: $AGENT_VERSION"
+echo "MODEL_CONFIG: $MODEL_CONFIG"
+
+# Default to use Hint
+if [ -z "$USE_HINT_TEXT" ]; then
+  export USE_HINT_TEXT=true
+fi
+echo "USE_HINT_TEXT: $USE_HINT_TEXT"
+EVAL_NOTE="$AGENT_VERSION"
+# if not using Hint, add -no-hint to the eval note
+if [ "$USE_HINT_TEXT" = false ]; then
+  EVAL_NOTE="$EVAL_NOTE-no-hint"
+fi
+
+# poetry run python ./opendevin/core/main.py -i 10 -t "tell me the usa's president using google search" -c BrowsingAgent -m gpt-4o-2024-05-13
+
+# exit 0
+
+COMMAND="poetry run python evaluation/webarena/run_infer.py \
+  --agent-cls $AGENT \
+  --llm-config $MODEL_CONFIG \
+  --max-iterations 15 \
+  --max-chars 10000000 \
```

Comment:
qq: `10000000` is a random number?

Reply:
yeah, it's just there to not truncate anything